test/e2e/framework/pod/utils.go

```diff
@@ -338,3 +338,7 @@ func IsPodOnCgroupv2Node(f *framework.Framework, pod *v1.Pod) bool {
 	}
 	return len(out) != 0
 }
+
+func RoundUp(in int64) int64 {
+	return (in/10 + 1) * 10
+}
```

Comment:
I know there are a million ways to do this, but this is how @hshiina did it in https://github.com/kubernetes/kubernetes/pull/127192/commits/9fc49b6efa23620b0edda777f43946f11bb75749 which struck me as nice and simple :)

Reply:
What is the right rounding factor here? https://github.com/kubernetes/kubernetes/commit/9fc49b6efa23620b0edda777f43946f11bb75749 uses `10`, but that's operating on MilliCPU quantities, whereas here this is the final quota string written to the cgroup. https://github.com/opencontainers/cgroups/blob/2a61babab3b079d757796fb543bf91400d15d4b9/systemd/common.go#L303-L310 is using a factor of `10,000`, but I think the errors we're seeing are rounding to `1,000`? I can't figure out how we get to 1000 though.

Reply:
I'll update this to use MilliCPU quantities in which case I think we can use 10?

Reply:
so my understanding of what is going on here is pretty shaky, but I updated this so that we round the millicpu quantity before converting it to the quota string, in which case I'm assuming the rounding factor should be 10?

Reply:
I think this is doing the right thing, added a log statement to a local run to see what was happening and I see it being rounded correctly:

```
expected cpu limits: original 1500 100000, rounded 2000 100000
```